src/components/video-player/video-player.test.js

```diff
@@ -12,7 +12,7 @@ const mockSettings = {
 
 const movie = testMovies[0];
 
-it(`should correnctly render VideoPlayer component`, () => {
+it(`should correctly render VideoPlayer component`, () => {
```

Comment:
после should лучше ставить глагол везде, чтобы хорошо читалось и понималось

```suggestion
it(`should render VideoPlayer component correctly`, () => {
```